farmbot_celery_script/config/config.exs

```diff
@@ -7,3 +7,5 @@ else
   config :farmbot_celery_script, FarmbotCeleryScript.SysCalls,
     sys_calls: FarmbotCeleryScript.SysCalls.Stubs
 end
+
+import_config "../../global_configs.exs"
```

Comment:
This is almost always a bad idea. The reason all of the apps are separate and not an umbrella is to better separate them.

See the [Poncho projects](https://embedded-elixir.com/post/2017-05-19-poncho-projects/) post